src/rns/index.ts

```diff
@@ -61,7 +61,7 @@ async function precache (eth?: Eth): Promise<void> {
   await fetchEventsForService(eth, 'rns.placement', simplePlacementsContractAbi as AbiItem[], dataQueuePusher)
 
   // We need to sort the events in order to have valid
-  eventsDataQueue.sort((a, b): number => {
+  const sortedQueue = eventsDataQueue.sort((a, b): number => {
     // First by block number
     if (a.blockNumber - b.blockNumber !== 0) return a.blockNumber - b.blockNumber
```

Comment:
I would prefer this to be:

`if (a.blockNumber !== b.blockNumber) return a.blockNumber - b.blockNumber`